packages/katello/katello/backup.rb

```diff
@@ -324,7 +324,14 @@ def generate_metadata
     end
 
     def create_pulp_data_tar
-      run_cmd("tar --selinux --create --file=#{File.join(@dir, 'pulp_data.tar')} --exclude=var/lib/pulp/katello-export --listed-incremental=#{File.join(@dir, '.pulp.snar')} --transform 's,^,var/lib/pulp/,S' -S *")
+      @archive_size = @options[:split_pulp_tar] || "0"
+      @split_pulp_tar_script = default_split_pulp_tar_script
```

Comment:
unless I'm missing something, I don't see a reason to make these instance variables with `@`

Reply:
I fear I'm not much of a ruby programmer.  I've not yet gotten the hang of when `@` makes sense.  I'll drop it from there.